src/crimson/os/seastore/omap_manager/btree/string_kv_node_layout.h

```diff
@@ -1341,10 +1241,9 @@ class StringKVLeafNodeLayout {
     auto iter = iter_begin();
     auto iter2 = rhs.iter_begin();
     while (iter != iter_end()) {
-      if(iter->get_node_key() != iter2->get_node_key() ||
-	       iter->get_node_val() != iter2->get_node_val() ||
-         iter->get_string_val() != iter2->get_string_val()){
-	      return false;
+      if(iter->get_key() != iter2->get_key() ||
+         iter->get_val() != iter2->get_val()) {
```

Comment:
get_val() return bufferlist,  I can't find bufferlist operator!= definition, should it be get_string_val() here?

Reply:
get_val() returns the mapped value for the key at iter, which happens to be a bufferlist.  operator!= is defined for bufferlists in include/buffer.h.